config/elasticsearch/mappings/classified_listings.json

```diff
@@ -19,14 +19,18 @@
       }
     },
     "body_markdown": {
-      "type": "text"
+      "type": "text",
+      "copy_to": "classified_listing_search"
     },
     "bumped_at": {
       "type": "date"
     },
     "category": {
       "type": "keyword"
     },
+    "classified_listing_search": {
```

Comment:
I'm not the biggest fan of this name, but it seemed better than addressing the 5 fields it's combining (`body_markdown`, `location`, `slug`, `tags`, `title`) with something like `body_markdown_location_slug_tags_title`

Reply:
Given this will likely change in short order I'm fine with this name. Also, the name describes exactly what we are using it for so +1 for that!